doc/api.js

```diff
@@ -298,7 +298,21 @@ export function getDllVersion(callback) {}
  */
 export function getConnectedDevices(callback) {}
 
-
+/**
+ * Async function to get a list of all connected devices serialnumber.
+ *
+ * @example
+ * nrfjprogjs.getSerialNumbers(function(err, serialNumbers) {
+ *      if (err) throw err;
+ *      for (let i = 0; i < devices.length; i++) {
+ *          console.log(devices[i]);
+ *      }
```

Comment:
s/devices/serialNumbers/

Reply:
Fixed it when I copied it out to test, but forgot to fix in the source. Will fix.